src/types/functions/index.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Execution Context
+ */
+export interface FunctionExecutionContext {
+  execution_id: string,
+  inputs: Record<string, unknown>,
+  function: {
+    id: string,
+    callback_id: string,
+    title: string,
+    description: string,
+    type: string,
+    input_parameters: [],
+    output_parameters: [],
```

Comment:
What are the item types in the arrays here? if only `any` works, we can make it `any[]` for now.